presto-sql-function/src/main/java/com/facebook/presto/sqlfunction/AbstractSqlInvokedFunctionNamespaceManager.java

```diff
@@ -127,6 +127,11 @@ public final FunctionMetadata getFunctionMetadata(FunctionHandle functionHandle)
         return metadataByHandle.getUnchecked((SqlFunctionHandle) functionHandle);
     }
 
+    protected void refreshFunctionsCache(QualifiedFunctionName functionName)
```

Comment:
Where is this used?

Reply:
It is used by subclasses in which `createFunction` is implemented. This method can be called to update the functions cache on a new function or a new version fo a function is created.

Reply:
Yes, it is in `presto-facebook` now that we have remove the plugin commit from the PR.